apps/prairielearn/src/pages/studentGradebook/studentGradebook.ts

```diff
@@ -3,11 +3,21 @@ import asyncHandler from 'express-async-handler';
 
 import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
 
+import logPageView from '../../middlewares/logPageView.js';
+
 import { StudentGradebookRowSchema, StudentGradebook } from './studentGradebook.html.js';
 
 const sql = loadSqlEquiv(import.meta.url);
 const router = Router();
 
+router.use([
+  function (req, res, next) {
+    res.locals.navSubPage = 'gradebook';
+    next();
+  },
```

Comment:
Ditto, let's move this into the template if possible.

Reply:
Moved this directly into the Navbar template in [15771e5](https://github.com/PrairieLearn/PrairieLearn/pull/10906/commits/15771e55e6dd9050a1d39214df8424edc073a18f).